internal/controller/policyserver_controller_test.go

```diff
@@ -263,7 +263,9 @@ var _ = Describe("PolicyServer controller", func() {
 			})))
 		})
 
-		It("should create the policy server configmap with the assigned policies", func() {
+		FIt("should create the policy server configmap with the assigned policies", func() {
```

Comment:
I think the `FIt` is a leftover from your development time, you should go back to `It`

Reply:
Woops.. rebased.